elements/pl-multiple-choice/pl-multiple-choice.py

```diff
@@ -28,17 +33,39 @@ def categorize_options(element):
             else:
                 incorrect_answers.append(answer_tuple)
             index += 1
+
+    file_path = pl.get_string_attrib(element, 'external-json', EXTERNAL_JSON_DEFAULT)
+    if file_path is not EXTERNAL_JSON_DEFAULT:
+        correct_attrib = pl.get_string_attrib(element, 'external-json-correct-key', EXTERNAL_JSON_CORRECT_KEY_DEFAULT)
+        incorrect_attrib = pl.get_string_attrib(element, 'external-json-incorrect-key', EXTERNAL_JSON_INCORRECT_KEY_DEFAULT)
+        if pathlib.PurePath(file_path).is_absolute():
+            json_file = file_path
+        else:
+            json_file = pathlib.PurePath(data['options']['question_path']).joinpath(file_path)
+        try:
+            with open(json_file, mode='r', encoding='utf-8') as f:
+                obj = json.load(f)
+                for text in obj.get(correct_attrib, []):
+                    correct_answers.append((index, True, text))
+                    index += 1
+                for text in obj.get(incorrect_attrib, []):
+                    incorrect_answers.append((index, False, text))
+                    index += 1
+        except FileNotFoundError:
+            raise Exception(f'JSON answer file: "{json_file}" could not be found')
```

Comment:
```suggestion
            raise Exception(f'JSON answer file: "{file_path} could not be found')
```

We want to display the path here and not the actual file object?

Reply:
Oh, this is to also accommodate relative paths. If `file_path` is abs, then `json_file` is the same.
But when a relative path is given, I think it will be more helpful to show the user what the resolved `data['options']['question_path'] + file_path` looks like for easier debug.

Reply:
Ah, okay, nevermind then.  This is fine.